services/drupal/config/sync/migrate_plus.migration.upgrade_d7_group_content_file.yml

```diff
@@ -22,7 +22,6 @@ source:
     name: etid
   d7_entity_type: file
   d8_entity_type: media
-  use_og_table: true
```

Comment:
Actually this change needs to go the other direction.  I made the change to this file only, neglecting to change it in the module's original config.  I haven't really found a lot of value in trying to maintain the source config from the module, TBH.